completion_aggregator/management/commands/reaggregate_course.py

```diff
@@ -59,8 +60,9 @@ def handle(self, *args, **options):
         if options['all']:
             options['course_keys'] = BlockCompletion.objects.values_list('course_key').distinct()
         CourseEnrollment = compat.course_enrollment_model()  # pylint: disable=invalid-name
-        for course_str in options['course_keys']:
-            course = CourseKey.from_string(course_str)
+        for course in options['course_keys']:
+            if isinstance(course, six.string_types):
```

Comment:
Interesting, didn't expect anything but strings here.

Reply:
Yep, me neither. I found out the hard way that `BlockCompletion.objects.values_list('course_key').distinct()` actually returns a list of `CourseKey` objects.